plugins/scaffolder-backend/src/service/router.ts

```diff
@@ -241,6 +241,15 @@ export async function createRouter(
 
       res.status(201).json({ id: result.taskId });
     })
+    .get('/v2/tasks', async (req, res) => {
+      const userEntityRef = req.query.createdBy?.toString();
```

Comment:
`toString` here should probably be replaced with something else. Either it's already a string or we'll we get something unexpected. Might be that we need `[createdBy].flat()[0]` here?